### ISSUE TITLE
[1.11.2-2393-6.1.0-BETA-2468] Exception trying to process over a phase (Creeper explosion)

### ISSUE THREAD

**Issue by bhi-elliot** [SpongePowered/SpongeForge#1687]
The following came up in the server log. Not sure what caused it or if it is even Sponge's issue.
I'm sorry, this is all of the log that I have available.

I am currently running:

SpongeForge version: spongeforge-1.11.2-2393-6.1.0-BETA-2468
Forge version: forge-1.11.2-13.20.1.2414-universal
Java version: 1.8.0_131
Operating System: Windows 10
Plugins (37): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, SpongeAPI, SpongeForge, FoamFixCore, ccl-entityhook, AdventureMMO, BiblioCraft, Biomes O' Plenty, BiblioWoodBoP, BiblioWoodForestry, BuildCraft Lib, BuildCraft, BC Builders, BC Energy, BC Factory, BC Robotics, BC Silicon, BC Transport, CodeChicken Lib, FoamFix, IndustrialCraft 2, Just Enough Items, Forestry, FoxCore, FoxGuard, Iron Chest, LuckPerms, Not Enough Items, Nucleus, Project Core, Project Inventories, Project Portals, Project Worlds, WorldEdit

What is the expected result?
No error.

What is the current result?
Error:
```
           [Server thread/ERROR] [Sponge]: /**********************************************************************************************************************************************/
[09:02:21] [Server thread/ERROR] [Sponge]: /*                                                 Exception trying to process over a phase!                                                  */
[09:02:21] [Server thread/ERROR] [Sponge]: /**********************************************************************************************************************************************/
[09:02:21] [Server thread/ERROR] [Sponge]: /* PhaseContext :                                                                                                                             */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: CapturedExplosion                                                                                                          */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: CaptureExplosion{explosion=net.minecraft.world.Explosion@79c4c96c}                                                       */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: CapturedEntities                                                                                                           */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: CapturedEntitiesSupplier{Captured=0}                                                                                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: CapturedBlocks                                                                                                             */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: CapturedBlocksSupplier{Captured=0}                                                                                       */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: Source                                                                                                                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: EntityCreeper['Creeper'/474008, l='*****', x=-17.21, y=56.00, z=27.02]                                                   */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: Notifier                                                                                                                   */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: EntityPlayerMP['*****'/463633, l='*****', x=-13.16, y=56.00, z=30.09]                                                    */
[09:02:21] [Server thread/ERROR] [Sponge]: /*         - Name: Source                                                                                                                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*           Object: EntityCreeper['Creeper'/474008, l='*****', x=-17.21, y=56.00, z=27.02]                                                   */
[09:02:21] [Server thread/ERROR] [Sponge]: /* Stacktrace:                                                                                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /* java.lang.IllegalStateException: Expected to be capturing a block position during an explosion! Please analyze the current phase context.  */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.event.tracking.TrackingUtil.lambda$throwWithContext$3(TrackingUtil.java:479)                                  */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     java.util.Optional.orElseThrow(Unknown Source)                                                                                         */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.event.tracking.phase.general.ExplosionState.spawnEntityOrCapture(ExplosionState.java:292)                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.event.tracking.phase.general.GeneralPhase.spawnEntityOrCapture(GeneralPhase.java:364)                         */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.event.tracking.CauseTracker.spawnEntity(CauseTracker.java:652)                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.func_72838_d(WorldServer.java:2697)                                                                    */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.EntityHanging.func_70099_a(EntityHanging.java:240)                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.item.EntityPainting.func_110128_b(SourceFile:150)                                                                 */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.EntityHanging.func_70097_a(EntityHanging.java:188)                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.Explosion.func_77278_a(Explosion.java:786)                                                                         */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.func_72885_a(WorldServer.java:3079)                                                                    */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.triggerInternalExplosion(WorldServer.java:2676)                                                        */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.interfaces.entity.explosive.IMixinExplosive.detonate(IMixinExplosive.java:53)                                 */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.monster.EntityCreeper.redirect$onExplode$zfa003(SourceFile:707)                                                   */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.monster.EntityCreeper.func_146077_cc(SourceFile:248)                                                              */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.entity.monster.EntityCreeper.func_70071_h_(SourceFile:158)                                                               */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     org.spongepowered.common.event.tracking.TrackingUtil.tickEntity(TrackingUtil.java:158)                                                 */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.redirect$onCallEntityUpdate$zke000(WorldServer.java:2866)                                              */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.World.func_72866_a(World.java:4051)                                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:840)                                                                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.World.func_72870_g(World.java:1934)                                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.World.func_72939_s(World.java:6464)                                                                                */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2203)                                                                    */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:720)                                                            */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:384)                                                  */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:624)                                                            */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     net.minecraft.server.MinecraftServer.run(MinecraftServer.java:482)                                                                     */
[09:02:21] [Server thread/ERROR] [Sponge]: /*     java.lang.Thread.run(Unknown Source)                                                                                                   */
[09:02:21] [Server thread/ERROR] [Sponge]: /**********************************************************************************************************************************************/
 /**********************************************************************************************************************************************/
/*                                                 Exception trying to process over a phase!                                                  */
/**********************************************************************************************************************************************/
/* PhaseContext :                                                                                                                             */
/*         - Name: CapturedExplosion                                                                                                          */
/*           Object: CaptureExplosion{explosion=net.minecraft.world.Explosion@79c4c96c}                                                       */
/*         - Name: CapturedEntities                                                                                                           */
/*           Object: CapturedEntitiesSupplier{Captured=0}                                                                                     */
/*         - Name: CapturedBlocks                                                                                                             */
/*           Object: CapturedBlocksSupplier{Captured=0}                                                                                       */
/*         - Name: Source                                                                                                                     */
/*           Object: EntityCreeper['Creeper'/474008, l='*****', x=-17.21, y=56.00, z=27.02]                                                   */
/*         - Name: Notifier                                                                                                                   */
/*           Object: EntityPlayerMP['*****'/463633, l='*****', x=-13.16, y=56.00, z=30.09]                                                    */
/*         - Name: Source                                                                                                                     */
/*           Object: EntityCreeper['Creeper'/474008, l='*****', x=-17.21, y=56.00, z=27.02]                                                   */
/* Stacktrace:                                                                                                                                */
/* java.lang.IllegalStateException: Expected to be capturing a block position during an explosion! Please analyze the current phase context.  */
/*     org.spongepowered.common.event.tracking.TrackingUtil.lambda$throwWithContext$3(TrackingUtil.java:479)                                  */
/*     java.util.Optional.orElseThrow(Unknown Source)                                                                                         */
/*     org.spongepowered.common.event.tracking.phase.general.ExplosionState.spawnEntityOrCapture(ExplosionState.java:292)                     */
/*     org.spongepowered.common.event.tracking.phase.general.GeneralPhase.spawnEntityOrCapture(GeneralPhase.java:364)                         */
/*     org.spongepowered.common.event.tracking.CauseTracker.spawnEntity(CauseTracker.java:652)                                                */
/*     net.minecraft.world.WorldServer.func_72838_d(WorldServer.java:2697)                                                                    */
/*     net.minecraft.entity.EntityHanging.func_70099_a(EntityHanging.java:240)                                                                */
/*     net.minecraft.entity.item.EntityPainting.func_110128_b(SourceFile:150)                                                                 */
/*     net.minecraft.entity.EntityHanging.func_70097_a(EntityHanging.java:188)                                                                */
/*     net.minecraft.world.Explosion.func_77278_a(Explosion.java:786)                                                                         */
/*     net.minecraft.world.WorldServer.func_72885_a(WorldServer.java:3079)                                                                    */
/*     net.minecraft.world.WorldServer.triggerInternalExplosion(WorldServer.java:2676)                                                        */
/*     org.spongepowered.common.interfaces.entity.explosive.IMixinExplosive.detonate(IMixinExplosive.java:53)                                 */
/*     net.minecraft.entity.monster.EntityCreeper.redirect$onExplode$zfa003(SourceFile:707)                                                   */
/*     net.minecraft.entity.monster.EntityCreeper.func_146077_cc(SourceFile:248)                                                              */
/*     net.minecraft.entity.monster.EntityCreeper.func_70071_h_(SourceFile:158)                                                               */
/*     org.spongepowered.common.event.tracking.TrackingUtil.tickEntity(TrackingUtil.java:158)                                                 */
/*     net.minecraft.world.WorldServer.redirect$onCallEntityUpdate$zke000(WorldServer.java:2866)                                              */
/*     net.minecraft.world.World.func_72866_a(World.java:4051)                                                                                */
/*     net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:840)                                                                     */
/*     net.minecraft.world.World.func_72870_g(World.java:1934)                                                                                */
/*     net.minecraft.world.World.func_72939_s(World.java:6464)                                                                                */
/*     net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2203)                                                                    */
/*     net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:720)                                                            */
/*     net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:384)                                                  */
/*     net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:624)                                                            */
/*     net.minecraft.server.MinecraftServer.run(MinecraftServer.java:482)                                                                     */
/*     java.lang.Thread.run(Unknown Source)                                                                                                   */
/**********************************************************************************************************************************************/

```

**Comment by Coolfeather2** [comment]
SpongePowered/SpongeForge#1701

**Comment by phit** [comment]
blew up painting with a creeper no errors with spongeforge-1.12.2-2529-7.0.0-BETA-2753

**Comment by gabizou** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.
